cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -546,6 +546,70 @@ storage:
                 value: https://identity.zalando.com/.well-known/openid-configuration
               - name: ENABLE_INTROSPECTION
                 value: "true"
+{{ if ne .Cluster.Environment "production" }}
+          - name: tokeninfo-sandbox
+            image: registry.opensource.zalan.do/foundation/platform-iam-tokeninfo:2fca26c
+            ports:
+            - containerPort: 9022
+            livenessProbe:
+              httpGet:
+                path: /health
+                port: 9022
+              periodSeconds: 3
+              failureThreshold: 2
+            readinessProbe:
+              httpGet:
+                path: /health
+                port: 9022
+              periodSeconds: 3
+              failureThreshold: 2
+            resources:
+              requests:
+                cpu: 100m
+                memory: 20Mi
+            env:
+            - name: OPENID_PROVIDER_CONFIGURATION_URL
+              value: https://sandbox.identity.zalando.com/.well-known/openid-configuration
+            - name: ENABLE_INTROSPECTION
+              value: "true"
+            - name: ISSUER
+              value: "https://sandbox.identity.zalando.com"
+            - name: LISTEN_ADDRESS
+              value: ":9022"
+          - name: skipper-tokeninfo-bridge
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.10.243
+            args:
+            - skipper
+            - -address=:9023
+            - -inline-routes
+            - |
+              health: Path("/healthz") -> inlineContent("ok") -> <shunt>;
+              q2h: Path("/oauth2/introspect") && QueryParam("token")
+                -> queryToHeader("token", "Authorization", "Bearer %s")
+                -> setRequestHeader("X-Checking-JWT", "true")
+                -> <loopback>;
+              prod: Path("/oauth2/introspect") && Header("X-Checking-JWT", "true") && JWTPayloadAllKV("iss", "https://identity.zalando.com")
+                -> dropRequestHeader("Authorization") -> dropRequestHeader("X-Checking-JWT")
+                -> "http://127.0.0.1:9021/oauth2/introspect";
+              sandbox: Path("/oauth2/introspect") && Header("X-Checking-JWT", "true") && JWTPayloadAllKV("iss", "https://sandbox.identity.zalando.com")
+                -> dropRequestHeader("Authorization") -> dropRequestHeader("X-Checking-JWT")
+                -> "http://127.0.0.1:9022/oauth2/introspect";
+              fallback1: Path("/oauth2/introspect") && QueryParam("token") && Header("X-Checking-JWT", "true")
```

Comment:
to avoid conflict between prod<->fallback1 and sandbox<->fallback1, either remove `QueryParam("token")` from fallback1, or add to both prod and sandbox. If we expect to add more routes here in the future, then probably the latter way is better.

Reply:
Done, thanks.